misc/css_generator.py

```diff
@@ -11,18 +11,19 @@
 """
 
 # File the CSS code will be written in
-dest_file = "to_be_appended_into_innovation.css"
+dest_file = ".\\misc\\to_be_appended_into_innovation.css"

```

Comment:
I moved the to_be_appened_into_innovation.css to build in misc, so that is not uploaded to bga

Reply:
I guess this also assumes that you are all on windows...

Reply:
If there is a need I'm sure we can upgrade the script to detect file system and adjust accordingly

Reply:
Would `dest_file = "misc/to_be_appended_into_innovation.css"` work regardless of the system?

Reply:
Actually, given that the css_generator.py script is already in misc, if you run it from that directory won't it put it in misc?

Does `".\\misc\\to_be_appended_into_innovation.css"` make it so that it always ends up in misc regardless of which directory you run the script in?

Reply:
I'll try the /

I have been running the py by right clicking it and running it, which happens to open the terminal in the project space, so it would generate it there. hmm if you were navigating to the file, then it would error out when you run it. I will see if I can find a universal solution

Reply:
I remembered how to do py stuff. will be fixed in upcoming commit to this branch